README.md

```diff
@@ -1,2 +1,79 @@
 # dagpenger-inntekt-api
 Holder kopi av inntekter fra inntektskomponenten for regelkjøring av dagpenger-behov
+
+
+## Utvikling av applikasjonen
+
+For å kjøre enkelte av testene kreves det at Docker kjører.
+
+[Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+
+### Starte applikasjonen lokalt
+
+Applikasjonen har avhengigheter til Postgres som kan kjøres
+opp lokalt vha Docker Compose(som følger med Docker Desktop) 
+
+
+Starte Postgres: 
+```
+
+docker-compose -f docker-compose.yml up
+
+```
+Etter at containerene er startet kan man starte applikasjonen ved å kjøre main metoden.
+
+
+Stoppe Kafka og Postgres:
+
+```
+ctrl-c og docker-compose -f docker-compose.yml down 
+
+```
+
+### Tilgang til Postgres databasen
+
+For utfyllende dokumentasjon se [Postgres i NAV](https://github.com/navikt/utvikling/blob/master/PostgreSQL.md)
+
+#### Tldr
+
+Applikasjonen benytter seg av dynamisk genererte bruker/passord til database.
+For å koble seg til databasen må man genere bruker/passord(som varer i en time)
+på følgende måte:
+
+Installere [Vault](https://www.vaultproject.io/downloads.html)
+
+Generere bruker/passord: 
+
+```
+
+export VAULT_ADDR=https://vault.adeo.no USER=NAV_IDENT
+vault login -method=oidc
+
+
+```
+
+Preprod credentials:
+
+```
+vault read postgresql/preprod-fss/creds/dp-regel-api-preprod-admin
```

Comment:
Wrong path. Should be dp-inntekt-db-preprod?

Reply:
doh..